src/main/resources/application.properties

```diff
@@ -16,11 +19,13 @@ osis.scality.vault.secret-key=UEEu8tYlsOGGrgf4DAiSZD6apVNPUWqRiPG0nTB6
 osis.scality.vault.cache.listAccounts.disabled=false
 osis.scality.vault.cache.listAccounts.maxCapacity=1000
 osis.scality.vault.cache.listAccounts.expirationInMS=60000
+osis.scality.vault.cache.assumeRole.maxCapacity=1000
+osis.scality.vault.cache.assumeRole.expirationInMS=60000
 
-# S3 Config
-osis.scality.s3.endpoint=http://scality.osis.ose.vmware.com:31383
-#osis.scality.s3.access-key=094BC4MZSZMET634FXZ0
-#osis.scality.s3.secret-key=Gpyohi7gTe0YriLNhURixitKoZ8IVAsCPrDDZxh6
+# S3 Config - Vault endpoint and S3 port with super admin credentials
```

Comment:
Vault has `superAdmin` credentials. S3/Cloudserver doesn't understand superAdmin credentials. 
Can be a little confusing, mentioning Vault should be enough if using `superAdmin` Creds

Reply:
Ok

Reply:
Done in 9101950281bb1e469ee5c3cbe333dcaa4c9a3953